processor/spanmetricsprocessor/processor_test.go

```diff
@@ -698,6 +709,9 @@ func initSpan(span span, s ptrace.Span) {
 	s.Attributes().PutEmptySlice(arrayAttrName)
 	s.SetTraceID(pcommon.TraceID([16]byte{byte(42)}))
 	s.SetSpanID(pcommon.SpanID([8]byte{byte(42)}))
+	s.TraceState().FromRaw(otelTraceState{
+		pvalue: uint8(pval),
+	}.serialize())
```

Comment:
This is setting an invalid tracestate. The serialize returns a string like `p:3;r:0`, when it should actually be `ot=p:3;r:0`. Similarly, because the tests using this are (presumably) passing, that means the tracestate handling code is incorrect. The spec is [here](https://www.w3.org/TR/trace-context/#list). I'd recommend manually creating the tracestate strings for the tests